✨(playbook) add delete_previous playbook

jmaupetit opened the pull request:
## Purpose

Being able to remove an old previous stack is useful to lower k8s cluster load. 

## Proposal

This new playbook should be used with caution only for stacks that will never be switched back.

jmaupetit committed:
✨(playbook) add delete_previous playbook

Being able to remove an old previous stack is useful to lower k8s
cluster load. This new playbook should be used with caution only for
stacks that will never be switched back.

jmaupetit commented on delete_previous.yml:
```diff
@@ -0,0 +1,33 @@
+    - name: Check if apps_filter is defined
+      fail:
+        msg: "This project contains several apps, the `apps_filter` variable should be defined"
+      when: apps_filter is not defined and apps | length > 1
```
This task is used in 3 different playbooks, should we extract it in a new task set and import it? Ping @lunika

lunika replied:
I think you can, yes.

jmaupetit commented on CHANGELOG.md:
```diff
@@ -13,6 +13,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
   base cluster load
 - Ensure expected secrets exists before deploying an application
+- Add `delete_previous.yml` playbook to delete previous stacks (should be used
+  with caution)
```
Wrong section.

jmaupetit committed:
🚚(tasks) rename validate_configuration to check_configuration

For semantic consistency, we've renamed the validate_configuration tasks
set to check_configuration.

jmaupetit committed:
💡(tasks) add heading docs to check_configuration

We've also changed a bit log messages.

jmaupetit committed:
🎨(playbook) move check_apps_filter to a tasks set

Let's be DRY on our checkings!
